builder_test.go

```diff
@@ -310,26 +310,187 @@ func testBuilder(t *testing.T, when spec.G, it spec.S) {
 							t.Fatalf("Unexpected error:\n%s\n", err)
 						}
 						if s := cmp.Diff(metadata.Processes, []launch.Process{
+							{
+								Type:        "some-type",
+								Command:     "some-command",
+								Args:        []string{"some-arg"},
+								Direct:      true,
+								BuildpackID: "A",
+							},
+							{
+								Type:        "some-other-type",
+								Command:     "some-other-command",
+								Args:        []string{"some-other-arg"},
+								Direct:      true,
+								BuildpackID: "B",
+							},
```

Comment:
I'm confused as to why the expectation changed when the setup stayed the same - it looks like the processes are in reverse order now - was that intentional?

Reply:
I see that you marked this conversation as resolved. Is it clear?